ludwig/utils/audio_utils.py

```diff
@@ -176,7 +251,7 @@ def _short_time_fourier_transform(
     window_length_in_samp = get_length_in_samp(window_length_in_s, sampling_rate_in_hz)
     window_shift_in_samp = get_length_in_samp(window_shift_in_s, sampling_rate_in_hz)
     preprocessed_data_matrix = _preprocess_to_padded_matrix(
-        data, window_length_in_samp, window_shift_in_samp, zero_mean_offset=zero_mean_offset
+        data[0], window_length_in_samp, window_shift_in_samp, zero_mean_offset=zero_mean_offset
```

Comment:
I know this was in the last PR, but I'm curious– why does this change? Is it because `torchaudio.load` loads with an extra `batch_size` dim?

Reply:
Right, so basically, the soundfile package loads the audio as a 1D np.array but the torchaudio load function loads it as a 2D torch.Tensor with shape(1, n) so I had to index into the tensor to actually get the data here.